x-pack/plugins/alerting/server/routes/clone_rule.test.ts

```diff
@@ -90,6 +91,7 @@ describe('cloneRuleRoute', () => {
     created_at: mockedRule.createdAt,
     updated_at: mockedRule.updatedAt,
     id: mockedRule.id,
+    revision: 0, // TODO: Finalize clone rule behavior. Clone or reset revision?
```

Comment:
is that for us or for you?

Reply:
It's for no one! 🙂 Got an answer over in [this thread](https://github.com/elastic/kibana/pull/147398/files#r1106484105) -- a clone will reset revision to `0`, so I'll remove this TODO. 👍